### ISSUE TITLE
auto-connect on linux should only affect current user

### ISSUE THREAD

**Issue by TimoKramer** [mullvad/mullvadvpn-app#631]
Hi,

I have multiple accounts on my linux-notebook, one for business and one for private. I activated mullvad auto-connect on my private account and it worked as expected. But when I used my business-user it took me a while to figure out that mullvad is breaking my vpn connection to my company. 

mullvad auto-connected on my business-account, too, although I only set auto-connect on my private account. This configuration should only be set for the account running the process. I did not expect it to be set globally.

Best regards

**Comment by faern** [comment]
The only way we have any chance of promising safety and leak protection is by controlling networking of the entire computer. For basically every OS, networking setup is something that is done globally and not per user basis. We ensure the safety by controlling the DNS settings, the firewall and the routing table. All of which are global to the computer and not controllable per user in general.

Basically any VPN client makes the traffic of the entire computer go through the tunnel. Vanilla OpenVPN also does this by default.

Thanks for your concerns, but I have to close this issue since it's by design and also something we can't really do in any other practical way.
